2_process/src/do_lake_toha_tasks.R

```diff
@@ -53,13 +51,37 @@ create_lake_tasks <- function(task_df_fn, log_folder){
     } 
   )
   
-  # ---- combine into a task plan ---- #
+  ##-- Create the task plan --##
   
-  gif_task_plan <- scipiper::create_task_plan(
+  task_plan <- create_task_plan(
     task_names=tasks$site_id,
     task_steps=list(split_morphometry,
-      calculate_model_toha),
+                    calculate_model_toha),
     add_complete=FALSE,
-    final_steps=c(toha_step_name),
-    ind_dir='2_process/log')
+    final_steps=c(toha_step_name))
+  
+  ##-- Create the task remakefile --##
+  
+  task_makefile <- sprintf('2_%s_lake_tasks.yml', model_prefix)
+  create_task_makefile( 
+    task_plan=task_plan,
+    makefile=task_makefile,
+    include='remake.yml',
+    sources=c(...),
+    packages=c("purrr", "dplyr", "mda.lakes", "feather", "rLakeAnalyzer"),
+    final_targets = final_target,
+    as_promises = TRUE,
+    tickquote_combinee_objects = TRUE
+  )
+  
+  ##-- Build the tasks --##
+  
+  loop_tasks(task_plan = task_plan,
+             task_makefile = task_makefile,
+             num_tries = 1, 
```

Comment:
I assume you are using `loop_tasks` here for the parallelization and not for the retries (given `num_tries = 1` here). Makes sense.

Reply:
Yes, exactly. I don't want these to try more than once because there is no flaky web service or something similar involved. So, retrying a failed task likely won't change the outcome.